lib/internal/bootstrap_node.js

```diff
@@ -268,15 +268,29 @@
     var console;
     Object.defineProperty(global, 'console', {
       configurable: true,
-      enumerable: true,
+      enumerable: false,
```

Comment:
These changes need to be semver-major as this might break user code, although hopefully _very_ rarely.